speculation-rules/prerender/restriction-background-sync.tentative.https.html

```diff
@@ -30,8 +30,7 @@
   // periodicSync.register() waits until the permission is granted, which
   // is deferred during prerendering so the test would trivially pass without
   // the permission.
-  await test_driver.set_permission({name: 'periodic-background-sync'}, 'granted',
-                                    location.origin, location.origin);
```

Comment:
Was this test relying on the first `location.origin` being converted to a boolean? won't removing the second one change the test's behavior?

Reply:
Argument is invalid